### PR TITLE
Support heiman HS1MS

### PR DESCRIPTION
**Pull request by Lupino**
http://www.heiman.com.cn/product/110.html

### CODE REVIEW

**Comment by Koenkk** [comment]
Thanks, not related this this but could you try pairing this device to the latest zigbee2mqtt dev branch, does it pair successfully? (Interview succeeded)

**Comment by Lupino** [comment]
```
lmj@Menjun-311:zigbee2mqtt[dev]$ node index.js
zigbee2mqtt:info  2020-01-19 16:30:15: Logging to console and directory: '/Users/lmj/super-app/zigbee/zigbee2mqtt/data/log/2020-01-19.16-30-15'
zigbee2mqtt:info  2020-01-19 16:30:15: Starting zigbee2mqtt version 1.9.0-dev (commit #2e958f8)
zigbee2mqtt:info  2020-01-19 16:30:15: Starting zigbee-herdsman...
zigbee2mqtt:info  2020-01-19 16:30:17: zigbee-herdsman started
zigbee2mqtt:info  2020-01-19 16:30:17: Coordinator firmware version: '{"type":"zStack12","meta":{"transportrev":2,"product":0,"majorrel":2,"minorrel":6,"maintrel":3,"revision":20190619}}'
zigbee2mqtt:info  2020-01-19 16:30:17: Currently 0 devices are joined:
zigbee2mqtt:warn  2020-01-19 16:30:17: `permit_join` set to  `true` in configuration.yaml.
zigbee2mqtt:warn  2020-01-19 16:30:17: Allowing new devices to join.
zigbee2mqtt:warn  2020-01-19 16:30:17: Set `permit_join` to `false` once you joined all devices.
zigbee2mqtt:info  2020-01-19 16:30:17: Zigbee: allowing new devices to join.
zigbee2mqtt:info  2020-01-19 16:30:17: Connecting to MQTT server at mqtt://localhost
zigbee2mqtt:info  2020-01-19 16:30:17: Connected to MQTT server
zigbee2mqtt:info  2020-01-19 16:30:17: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'
zigbee2mqtt:info  2020-01-19 16:30:17: MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"version":"1.9.0-dev","commit":"2e958f8","coordinator":{"type":"zStack12","meta":{"transportrev":2,"product":0,"majorrel":2,"minorrel":6,"maintrel":3,"revision":20190619}},"log_level":"info","permit_join":true}'
zigbee2mqtt:warn  2020-01-19 16:30:29: Device '0x000d6f0014bb221a' left the network
zigbee2mqtt:info  2020-01-19 16:30:29: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"device_removed","message":"left_network","meta":{"friendly_name":"0x000d6f0014bb221a"}}'
zigbee2mqtt:info  2020-01-19 16:30:35: Device '0x000d6f0014bb221a' joined
zigbee2mqtt:info  2020-01-19 16:30:35: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"device_connected","message":{"friendly_name":"0x000d6f0014bb221a"}}'
zigbee2mqtt:info  2020-01-19 16:30:35: Starting interview of '0x000d6f0014bb221a'
zigbee2mqtt:info  2020-01-19 16:30:35: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"pairing","message":"interview_started","meta":{"friendly_name":"0x000d6f0014bb221a"}}'
zigbee2mqtt:info  2020-01-19 16:31:11: Successfully interviewed '0x000d6f0014bb221a', device has successfully been paired
zigbee2mqtt:info  2020-01-19 16:31:11: Device '0x000d6f0014bb221a' is supported, identified as: HEIMAN Smart Montion Sensor (HS1MS)
zigbee2mqtt:info  2020-01-19 16:31:11: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"pairing","message":"interview_successful","meta":{"friendly_name":"0x000d6f0014bb221a","model":"HS1MS","vendor":"HEIMAN","description":"Smart Montion Sensor","supported":true}}'
zigbee2mqtt:info  2020-01-19 16:31:11: Configuring '0x000d6f0014bb221a'
zigbee2mqtt:info  2020-01-19 16:31:16: Successfully configured '0x000d6f0014bb221a'
```

**Comment by Lupino** [comment]
Yes it Interview succeeded with the dev version

File: devices.js [hunk 1]
```diff
@@ -4348,6 +4348,21 @@ const devices = [
         fromZigbee: [fz.iaszone_occupancy_1],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['PIRSensor-EM'],
+        model: 'HS1MS',
+        vendor: 'HEIMAN',
+        description: 'Smart motion Sensor',
+        supports: 'occupancy',
+        fromZigbee: [fz.iaszone_occupancy_1],
```
Comment (Koenkk) [inline devices.js]: Battery reporting is configured below but there is no corresponding converter, is `battery_3V` missing from here?

Comment (Lupino) [reply devices.js]: I add battery_3V not found any other attribute report

**Comment by Koenkk** [comment]
This device already is supported by now: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L4734

**Pull request closed without merging**